Index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="Style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <h2 class="myLogo">My logo</h2>
+    <img class="burger" src="Assests/Union.jpg">
+  </nav>
+  <section class="sectionOne">
+    <h1 class="introTitle">I'm Kevin. Glad to see you!</h1>
+    <h3 class="textIntro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
```

Comment:
- [ ] This paragraph (e.g., "I'm a software ...") should use a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. Please fix this
- [ ] The font-size of `25px` is too much, what it should be is `16px` according to the Figma design.